tier2/repometrics/hooks/post_gen_project.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import time 
+
+# Get working directory and repometrics file
+current_dir = os.getcwd()
+repometrics_file = "repometrics.json"
+source_file = os.path.join(current_dir, repometrics_file)
+
+# Get parent directory, one level up from the current directory
+parent_dir = os.path.abspath(os.path.join(current_dir, '..'))
```

Comment:
I'm not an expert, but I believe this is a reasonable method, and doesn't open us up to path-traversal attacks, but maybe @IsaacMilarky can take a closer look.